stanford-earth/SE3_BLT
# Earth 515 news subs

# READY FOR REVIEW

# Summary
- New view for earth news sub pages.
- Deleted several old fields.

# Needed By (Date)
- end of sprint

# Steps to Test

1. checkout branch
2. import config
3. verify the view exists. its called `Earth News`


# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
